website/docs/components/form/layout/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Layout
+description: The Form layout components are used as structural components to organize and space the structure and spacing between elements in forms.
+caption: The Form layout components are used as structural components to organize and space the structure and spacing between elements in forms.
```

Comment:
```suggestion
caption: Form layout components create consistent structure and spacing for form elements.
```

Reply:
Note: Captions should be as brief as possible. Descriptions can be a bit longer though.